app/controllers/publishers/vacancies/base_controller.rb

```diff
@@ -37,15 +37,19 @@ def form_sequence
     )
   end
 
+  def redirect_to_job_summary
+    if vacancy.published?
+      redirect_to organisation_job_path(vacancy.id), success: t("publishers.vacancies.show.success")
+    else
+      redirect_to organisation_job_review_path(vacancy.id)
+    end
+  end
```

Comment:
This bit is called in 2 different places so extracted it into an easier to understand/follow method.